### PR TITLE
docs open poll

### PR DESCRIPTION
**Pull request by m4ra**
**Tasks**
- [ ] PR name contains story or task reference
- [x] Documentation (docs and inline)
- [ ] Tests (including n+1 and django_assert_num_queries where applicable)
- [ ] Changelog

### CODE REVIEW

File: docs/open_poll.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+Answers and Votes classes are updated and now they inherit from the new `GeneratedContentModel`. Answer has now as unique together `question`, `creator`, `content_id` depending on whether creator or content_id field exist.
+
+Permissions `allows_unregistered_users` are now also in place for enabling those unregistered users to interact with the open poll in the  - see the file [adhocracy4/polls/predicates.py](https://github.com/liqd/adhocracy4/blob/main/adhocracy4/polls/predicates.py).
+
+For submitting the open poll, we provide a checkbox for `agreed_terms_of_use` at the end of the poll, along with a [captcha written in React](https://github.com/liqd/adhocracy4/blob/main/adhocracy4/static/Captcha.jsx) to filter out robots and spam. For regular users the `agreed_terms_of_use` is set True when they are registering.
```
Comment (goapunk) [inline docs/open_poll.md:10]: The checkbox is unrelated to the unregistered users, it already existed before this. When registering on a+ you agree to the terms of use of a+ but not to the terms of use of the organisation you are participating in. So every time you participate in a project of a organisation you haven't participated before you will see the checkbox. Unregistered users just see it every time of course.

**Review by goapunk — COMMENTED** [review]
Looks good so far, just a small note about the terms of use

File: docs/open_poll.md (hunk 1 above)
Comment (m4ra) [reply docs/open_poll.md:10]: Thanks for clarifying!

**Review by goapunk — APPROVED** [review]
Thanks!

**Pull request merged**